engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/deployment/DeploymentPreparer.scala

```diff
@@ -0,0 +1,84 @@
+package pl.touk.nussknacker.k8s.manager.deployment
+
+import io.circe.syntax.EncoderOps
+import monocle.Iso
+import monocle.macros.GenLens
+import monocle.std.option._
+import pl.touk.nussknacker.engine.api.ProcessVersion
+import pl.touk.nussknacker.k8s.manager.K8sDeploymentManager.{labelsForScenario, objectNameForScenario, scenarioIdLabel, scenarioVersionAnnotation}
+import pl.touk.nussknacker.k8s.manager.K8sDeploymentManagerConfig
+import skuber.EnvVar.FieldRef
+import skuber.LabelSelector.IsEqualRequirement
+import skuber.apps.v1.Deployment
+import skuber.{Container, EnvVar, HTTPGetAction, LabelSelector, Pod, Probe, Volume}
+
+case class DeploymentPreparer(config: K8sDeploymentManagerConfig) {
```

Comment:
can it be normal class instead of case class? It is clasic service - not a bean. We don't want to pattern match it or check field by field if it equals with any other service

Reply:
done